ArgusDocker/simple/argus.properties

```diff
@@ -1,5 +1,10 @@
 # Default settings for unit and integration tests.
-build.property.persistence.unit=<provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>\n    <exclude-unlisted-classes>false</exclude-unlisted-classes>\n    <properties>\n      <property name="javax.persistence.schema-generation.database.action" value="drop-and-create-tables"/>\n      <property name="javax.persistence.jdbc.driver" value="org.apache.derby.jdbc.EmbeddedDriver"/>\n      <property name="javax.persistence.jdbc.url" value="jdbc:derby:memory:argus;create=true"/>\n      <property name="javax.persistence.jdbc.user" value="root"/>\n      <property name="javax.persistence.jdbc.password" value="root"/>\n      <property name="eclipselink.ddl-generation" value="drop-and-create-tables"/>\n      <property name="eclipselink.logging.level" value="SEVERE"/>\n      <property name="eclipselink.logging.parameters" value="true"/>\n      <property name="eclipselink.target-database" value="DERBY"/>\n      <property name="eclipselink.canonicalmodel.subpackage" value="unit"/>\n    </properties>
+#build.property.persistence.unit=<provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>\n#<exclude-unlisted-classes>false</exclude-unlisted-classes>\n    <properties>\n      <property name="javax.persistence.schema-generation.database.action" value="drop-and-create-tables"/>\n      <property name="javax.persistence.jdbc.driver" value="org.apache.derby.jdbc.EmbeddedDriver"/>\n      <property name="javax.persistence.jdbc.url" value="jdbc:derby:memory:argus;create=true"/>\n      <property name="javax.persistence.jdbc.user" value="root"/>\n      <property name="javax.persistence.jdbc.password" value="root"/>\n      <property name="eclipselink.ddl-generation" value="drop-and-create-tables"/>\n      <property name="eclipselink.logging.level" value="SEVERE"/>\n      <property name="eclipselink.logging.parameters" value="true"/>\n      <property name="eclipselink.target-database" value="DERBY"/>\n      <property name="eclipselink.canonicalmodel.subpackage" value="unit"/>\n    </properties>
+build.property.persistence.driver=com.mysql.jdbc.Driver
```

Comment:
We should revert this and use the method that has already been merged into kiwigrid-develop to set jpa properties